relayer/chains/substrate/event_parser.go

```diff
@@ -108,26 +113,27 @@ func (scp *SubstrateChainProcessor) parseEvent(
 			eventType = intoIBCEventType(eType)
 
 		case ReceivePacket, WriteAcknowledgement:
-
+			// TODO (QUESTION): the cosmos parser uses the packet accumulator for other events like the SendPacket,
+			//  is there a reason that doesn't work for us? Can receive packet
```

Comment:
I'm gonna defer this question to @agouin as well

Reply:
@jtieri Same as previous point, This is for composable team to check while testing via ibctest